doc/classes/String.xml

```diff
@@ -549,34 +617,68 @@
 		</method>
 		<method name="replace" qualifiers="const">
 			<return type="String" />
-			<argument index="0" name="what" type="String" />
-			<argument index="1" name="forwhat" type="String" />
+			<argument index="0" name="old" type="String" />
+			<argument index="1" name="new" type="String" />
+			<argument index="2" name="count" type="int" default="PackedInt32Array()" />
```

Comment:
That default value is bogus.

Reply:
Indeed. I thought doctool would fix that automatically. Will fix in a future PR.